book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :author, :title
+  attr_reader :rentals
+
```

Comment:
Please add a method for adding rentals in the `Book` class and the book itself should be sent as a parameter to create the new rental achieving the `has-many` association. You may learn more about this method [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship).